lib/lib-dynamodb/README.md

```diff
@@ -0,0 +1,175 @@
+# @aws-sdk/lib-dynamodb
+
+[![NPM version](https://img.shields.io/npm/v/@aws-sdk/lib-dynamodb/latest.svg)](https://www.npmjs.com/package/@aws-sdk/lib-dynamodb)
+[![NPM downloads](https://img.shields.io/npm/dm/@aws-sdk/lib-dynamodb.svg)](https://www.npmjs.com/package/@aws-sdk/lib-dynamodb)
+
+## Overview
+
+The document client simplifies working with items in Amazon DynamoDB by
+abstracting away the notion of attribute values. This abstraction annotates native
+JavaScript types supplied as input parameters, as well as converts annotated
+response data to native JavaScript types.
+
+## Marshalling Input and Unmarshalling Response Data
+
+The document client affords developers the use of native JavaScript types
+instead of `AttributeValue`s to simplify the JavaScript development
+experience with Amazon DynamoDB. JavaScript objects passed in as parameters
+are marshalled into `AttributeValue` shapes required by Amazon DynamoDB.
+Responses from DynamoDB are unmarshalled into plain JavaScript objects
+by the `DocumentClient`. The `DocumentClient` does not accept
+`AttributeValue`s in favor of native JavaScript types.
+
+|          JavaScript Type          | DynamoDB AttributeValue |
+| :-------------------------------: | ----------------------- |
+|              String               | S                       |
+|          Number / BigInt          | N                       |
+|              Boolean              | BOOL                    |
+|               null                | NULL                    |
+|               Array               | L                       |
+|              Object               | M                       |
+|                Set                | BS / NS / SS            |
```

Comment:
nit: Specify each of `BS/NS/SS`'s corresponding JS Type

Reply:
```suggestion
|   Set\<Uint8Array, Blob, ...\>    | BS                      |
|       Set\<Number, BigInt\>       | NS                      |
|           Set\<String\>           | SS                      |
```